docs/assemblies/assembly-prometheus-metrics-proxy.adoc

```diff
@@ -0,0 +1,79 @@
+// file included in the following:
+//
+// kroxylicious-operator/index.adoc
+
+[id='assembly-prometheus-metrics-proxy-{context}']
+= Proxy Metrics
```

Comment:
Not necessarily part of this PR but I think it might be worth talking about how to get an aggregate view of the proxy process and a view of a whole virtual cluster. e.g. total network throughput and then throughput per virtual cluster. As they are likely to the high level questions people operating the proxy are likely to ask.